common/src/main/kotlin/streams/service/Topics.kt

```diff
@@ -56,29 +65,37 @@ object TopicUtils {
     }
 
     inline fun <reified T: Throwable> validate(topics: Topics) {
-        val crossDefinedTopics = (topics.cdcSourceIdTopics + topics.cdcSchemaTopics).intersect(topics.cypherTopics.keys)
         val exceptionStringConstructor = T::class.constructors
                 .first { it.parameters.size == 1 && it.parameters[0].type.javaType == String::class.java }!!
+        val crossDefinedTopics = topics.allTopics()
```

Comment:
I wanted to find a shorter way but `(topics - topics.distinct()).distinct()` didn't work.

so this might be the best: `topics.groupBy({it},{1}).filterValues{ it.sum() > 1 }.keys`

Reply:
ok